app/Filament/Resources/UserResource/Pages/ViewUser.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Filament\Resources\UserResource\Pages;
+
+use App\Filament\Resources\UserResource;
+use App\Site\Models\User;
+use Filament\Actions;
+use Filament\Resources\Pages\ViewRecord;
+
+class ViewUser extends ViewRecord
+{
+    protected static string $resource = UserResource::class;
+
+    public static function getNavigationLabel(): string
+    {
+        return __('Details');
+    }
```

Comment:
Would it be possible to get an explicit "None" to show up if the user has no roles?
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/25b7f7df-3173-4028-a3bd-7cad5e47aa94)

Reply:
hidden if empty in latest